content/en/security_platform/cloud_siem/security_home.md

```diff
@@ -14,41 +14,41 @@ further_reading:
 
 The [Security Home][1] page is an entry point to your Cloud SIEM (Security Information and Event Management) environment. Readily access logs that are analyzed to detect threats, signals generated from [default][2] or [custom][3] log detection rules, and threats that require attention and remediation. See the status of logging sources and configure new sources in one location.
 
-{{< img src="security_platform/security_monitoring/security_home/overview.png" alt="Security Home" width="75%">}}
+{{< img src="security_platform/security_monitoring/security_home/cloud_siem_home.png" alt="The Cloud SIEM Home page showing the number of logs analyzed, signals, and high/critical signals, along with AWS and Azure as log sources" >}}
 
 ## Analyzed logs
 
-Get an overview of analyzed logs across all sources for a quick audit, or select **Logs Analyzed** to see a list of your analyzed logs in [Logs Explorer][2] for a more granular investigation. In Logs Explorer, filter by [log facets][3] or [aggregate your logs][4] to examine further into analyzed logs.
+Get an overview of analyzed logs across all sources for a quick audit, or select **Logs Analyzed** to see a list of your analyzed logs in [Log Explorer][4] for a more granular investigation. In Log Explorer, filter by [log facets][5] or [aggregate your logs][6] to examine further into analyzed logs.
 
-## Signals generated
+## Signals
 
-Analyze the number of signals generated and the number of detection rules that are triggering signals, or select **Signals Generated** to filter by log detection rules in the [Signals Explorer][5].
+Analyze the number of signals generated and the number of detection rules that are triggering signals, or select **Signals** to filter by log detection rules in the [Signals Explorer][7].
 
-In the Signals Explorer, click on any rule with a generated signal to inspect the signal further. Select the **Message** tab to learn how to [triage and respond to a generated signal][6], or click on any of the event attributes listed at the top of the panel or in the **Event Attributes** tab to [filter signals by attributes][6].
+In the Signals Explorer, click on any rule with a generated signal to inspect the signal further. Select the **Rule Details** tab to see rule's goals and strategies, and how to triage and respond to the [security signal][8], or click on any of the event attributes listed at the top of the panel or in the **Event Attributes** tab to [filter signals by attributes][8].
```

Comment:
Hey @jmassey-datadog! Could you confirm that the "Events Attributes" tab is no longer in the Signal panel? I'm not seeing it there.

Reply:
Spoke IRL: Pull out the paragraph on Signals Explorer because we have that information in a separate page now.